### PR TITLE
 Added sample query documentation for CONTAINS filter

### PR DESCRIPTION
**Pull request by varun-raj**
I've added a sample query for `CONTAINS` filter.

## Checklist
 - [ ]  A link to the issue/user story that the pull request relates to
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [x]  Documentation - any JSDoc, website, or Stackoverflow answers?

## Issue/User story
https://github.com/hyperledger/composer/issues/2836

### CODE REVIEW

**Comment by varun-raj** [comment]
@davidkel Can you please look into it?

File: packages/composer-website/jekylldocs/reference/query-language.md [hunk 1]
```diff
@@ -64,6 +64,20 @@ query Q17 {
 }
 ```
 
+#### Sample Contains queries
+
+The `CONTAINS` filter is used to search a array field in a node. The below query returns all the drivers who earned the punchual and steady-driving badges. Considering that the badges is of array type in driver participant.
```
Comment (nklincoln) [inline packages/composer-website/jekylldocs/reference/query-language.md]: typo on 'punchual' (assume you intended 'punctual')

Comment (varun-raj) [reply packages/composer-website/jekylldocs/reference/query-language.md]: Oops. Yeah, and updated it :)

**Comment by nklincoln** [comment]
thanks for the updated file - at the moment the [DCO sign off](https://github.com/probot/dco#how-it-works) is missing and we can't merge it until added. 

Once set up, it is a `-s` flag on the commit action to add the information. It is my understanding that it has to be applied on each commit, so it may be necessary to squash the current set of commits or reapply your changes to the current master branch and resubmit a new PR (feel free to assign the PR review to me if you go for this option)

Thanks again

**Comment by varun-raj** [comment]
@nklincoln I've updated the commits with sign, can you please check it now?

**Pull request merged**